packages/squiggle-lang/src/rescript/utility/Sparklines.res

```diff
@@ -0,0 +1,28 @@
+// Port of Sindre Sorhus' Sparkly to Rescript
+// reference implementation: https://github.com/sindresorhus/sparkly
+// Omitting rgb "fire" style, so no `chalk` dependency
+// Omitting: NaN handling, special consideration for constant data.
+
+let create = (
+  numbers: array<float>,
```

Comment:
Now that I think about it, it could be better to call these ``heights``; "numbers" is kinda meaningless (might as well call float, at that point)

Reply:
good point